KsApi/MockService.swift

```diff
@@ -1744,6 +1753,12 @@
     func fetchDiscovery_combine(paginationUrl _: String) -> AnyPublisher<DiscoveryEnvelope, ErrorEnvelope> {
       Empty(completeImmediately: false).eraseToAnyPublisher()
     }
+
+    func exchangeTokenForOAuthToken(params _: OAuthTokenExchangeParams)
+      -> AnyPublisher<OAuthTokenExchangeResponse, ErrorEnvelope> {
+      // TODO: Implement for testing.
+      Empty(completeImmediately: false).eraseToAnyPublisher()
```

Comment:
I need to come back and evaluate what of this flow I can test. It's all fairly tightly integrated with Apple's authentication controller. I could add tests just for these new endpoints, but since network tests go against these mocks anyways, it doesn't seem super valuable to me...

Reply:
@ifosli @scottkicks I'm open to suggestions for what seems worthwhile to test here.